fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/WeightCheckRequestHandlerSpec.groovy

```diff
@@ -52,6 +54,8 @@ class WeightCheckRequestHandlerSpec extends Specification {
         groupByQuery = Mock(GroupByQuery)
         groupByQuery.getInnermostQuery() >> groupByQuery
         response = Mock(WeightCheckResponseProcessor)
+        RequestLog.getId()
+        RequestLog.record(new BardQueryInfo(""))
```

Comment:
Do you want to perhaps add a test to show the value incrementing in here somewhere?

Reply:
@michael-mclawhorn Sure